src/DwitTech.AccountService.WebApi/Controllers/UserController.cs

```diff
@@ -99,6 +99,26 @@ public async Task<IActionResult> ChangePassword([FromBody] ChangePasswordModel p
         }
 
 
+        [HttpPut]
+        [Route("editaccount")]
+        public async Task<IActionResult> EditAccount([FromBody] EditRequestDto editRequest)
+        {
+            try
+            {
+                string authHeader = Request.Headers["Authorization"];
+                var editResult = await _userService.EditAccount(authHeader, editRequest);
+                if (editResult)
+                    return Ok(editResult);
```

Comment:
I'm confused as to why you'd want to return True to the user as a response after an editing his/her account.